codenames-common/src/main/resources/application.yml

```diff
@@ -1,18 +1,43 @@
 
 
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/codenames
```

Comment:
substitute that with env variables with default values (like line 25-26)

Reply:
Yes, it's a good idea